src/components/OpportunitiesContent/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+import "./index.scss";
+import { useTranslation } from "react-i18next";
+import OpportunitiesCard from "../OpportunitiesCard";
+import SearchBar from "../SearchBar";
+import JobsTrainingButtons from "../JobsTrainingButtons";
+
+const JobsTrainingContent = () => {
+  const { t } = useTranslation();
+  const [cardsList, setCardsList] = useState([
```

Comment:
add a comment above here explaining the diff too

Reply:
done :D